sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -238,7 +199,1702 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Compile>
     <Compile Include="**\*.cs" Exclude="**\obj\**\*;**\bin\**\*;**\*.Designer.cs;TestCases\**\*.cs;TestCasesForRuleFailure\**\*.*" />
-    <None Include="TestCases\**\*.*">
+    <None Include="TestCases\AbstractTypesShouldNotHaveConstructors.cs">

```

Comment:
I think the following many lines should be removed.

Reply:
Crap... indeed